tests/integration/conftest.py

```diff
@@ -246,6 +282,66 @@ def single_passing_test_step_for_mk8s(_zigzag_runner_factory, mk8s_config_file,
     return zz_runner
 
 
+@pytest.fixture(scope='session')
+def custom_build_url_config_file(tmpdir_factory):
+    """A config for zigzag used by the ASC team
```

Comment:
Description is vague.

Reply:
addressing in fb1987cbd8f4107ef7aba96c807d6cadfa7074dd